apps/judicial-system/web/src/routes/Prison/IndictmentOverview/IndictmentOverview.strings.ts

```diff
@@ -26,4 +26,36 @@ export const strings = defineMessages({
     defaultMessage: 'Dómur',
     description: 'Titill á Dómur hluta',
   },
+  punishmentTypeTitle: {
+    id: 'judicial.system.core:indictment_overview.punishmentType_title',
+    defaultMessage: 'Refsitegund',
+    description: 'Titill á Refsitegund',
+  },
+  imprisonment: {
+    id: 'judicial.system.core:indictment_overview.punishmentType.imprisonment',
```

Comment:
And same here, consider changing these strings in to this format for consistency. Also feel free to challenge it though if you think we should consider doing it differently ☺️

```
  punishmentTypeImprisonment: {
    id: 'judicial.system.core:indictment_overview.punishment_type_imprisonment',
```